fairlearn/reductions/_moments/bounded_group_loss.py

```diff
@@ -36,6 +38,8 @@ def default_objective(self):
 
     def load_data(self, X, y, **kwargs):
         """Load data into the moment object."""
+        if kwargs.get(_KW_CONTROL_FEATURES) is not None:
```

Comment:
Let's drop this. If you're worried about using unsupported `kwargs` (a good worry), we should change the signature of `load_data` to
```python
    def load_data(self, X, y, *, sensitive_features):
```

Reply:
Done